### ISSUE TITLE
Arnold Won't Render With UVMap Missing

### ISSUE THREAD

**Issue by AlexeyAdamitsky** [lunadigital/btoa#16]
Blender 2.91.2
BtoA Dev Branch cb259ea
Arnold SDK 6.1.0
Windows 10 2004

It's a strange case and I can't figure out why it's happening.

The error code:
```
Python: Traceback (most recent call last):
  File "C:\Users\...\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\btoa-feature-nodes\engine.py", line 96, in update
    node = btoa.generate_aipolymesh(ob, depsgraph)
  File "C:\Users\...\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\btoa-feature-nodes\btoa\bridge.py", line 62, in generate_aipolymesh
    mesh = bake_geometry(ob, depsgraph)
  File "C:\Users\...\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\btoa-feature-nodes\btoa\bridge.py", line 54, in bake_geometry
    mesh.calc_tangents()
RuntimeError: Error: Tangent space computation needs an UVMap, "(null)" not found, aborting

location: <unknown location>:-1
```

I use Extra Objects addon to create geometry.
![image](https://user-images.githubusercontent.com/64398828/107881995-01701700-6ef8-11eb-8f67-49f6d22e78e6.png)

In my case I used it to create round cube.
![image](https://user-images.githubusercontent.com/64398828/107882016-1d73b880-6ef8-11eb-8113-30231036800a.png)

Arnold won't render with this object/geometry in scene for me. I attached the file. If you remove delete round cube Arnold will render just fine.

![image](https://user-images.githubusercontent.com/64398828/107882054-56139200-6ef8-11eb-9783-bc2f5b7a9274.png)

Test scene:
[Arnold_Test_Scene3_001.zip](https://github.com/lunadigital/btoa/files/5978029/Arnold_Test_Scene3_001.zip)

**Comment by osiriswrecks** [comment]
Can you post the text from the console window?

**Comment by AlexeyAdamitsky** [comment]
> Can you post the text from the console window?

Of course. I updated the original post.

**Comment by AlexeyAdamitsky** [comment]
Ok, I figured it out. If objects are missing UVMap it'll produce this error. It happens with any objects. Just remove UVMap and Arnold will give this error.

**Comment by osiriswrecks** [comment]
Fixed in commit d66bf5a
